browser/main/modals/PreferencesModal/UiTab.js

```diff
@@ -182,22 +183,11 @@ class UiTab extends React.Component {
                 onChange={(e) => this.handleUIChange(e)}
                 ref='uiLanguage'
               >
-                <option value='sq'>{i18n.__('Albanian')}</option>
-                <option value='zh-CN'>{i18n.__('Chinese (zh-CN)')}</option>
-                <option value='zh-TW'>{i18n.__('Chinese (zh-TW)')}</option>
-                <option value='da'>{i18n.__('Danish')}</option>
-                <option value='en'>{i18n.__('English')}</option>
-                <option value='fr'>{i18n.__('French')}</option>
-                <option value='de'>{i18n.__('German')}</option>
-                <option value='hu'>{i18n.__('Hungarian')}</option>
-                <option value='ja'>{i18n.__('Japanese')}</option>
-                <option value='ko'>{i18n.__('Korean')}</option>
-                <option value='no'>{i18n.__('Norwegian')}</option>
-                <option value='pl'>{i18n.__('Polish')}</option>
-                <option value='pt-BR'>{i18n.__('Portuguese (Brazil)')}</option>
-                <option value='pt-PT'>{i18n.__('Portuguese (Portugal)')}</option>
-                <option value='ru'>{i18n.__('Russian')}</option>
-                <option value='es-ES'>{i18n.__('Spanish')}</option>
+                {
+                  languages.map((language) => {
```

Comment:
`()` for a single argument and `return` are unnecessary in arrow functions.
please rewrite like below:
```jsx
languages.map(language =>
  <option value={language.locale} key={language.locale}>{i18n.__(language.name)}</option>)
```